package-res/reportviewer/paginationTemplate.html

```diff
@@ -238,6 +240,21 @@
     );
   }
 
+  function toLocaleStringSupportsLocales() {
+  var number = 0;
+  try {
+    number.toLocaleString('i');
+  } catch (e) {
+    return e.name === 'RangeError';
```

Comment:
Put your old regexp-based solution into the catch block here. This way, if "toLocaleString" is not available, the user will get at least a result even though it is not in the correct locale.